src/Component/RequestBuilder/FacetSet.php

```diff
@@ -117,19 +117,30 @@ public function addFacetField($request, $facet)
             'facet.field',
             $this->renderLocalParams(
                 $field,
-                ['key' => $facet->getKey(), 'ex' => $facet->getExcludes()]
+                ['key' => $facet->getKey()
+                  , 'ex' => $facet->getExcludes()
```

Comment:
These lines violate the coding standard, see https://github.styleci.io/analyses/q5l0Ey?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link